packages/agoric-cli/lib/start.js

```diff
@@ -112,9 +112,9 @@ export default async function startMain(progname, rawArgs, powers, opts) {
 
   let agSolo;
   if (opts.sdk) {
-    agSolo = path.resolve(__dirname, '../../cosmic-swingset/bin/ag-solo');
+    agSolo = path.resolve(__dirname, '../../solo/src/entrypoint.cjs');
   } else {
-    agSolo = `/usr/src/dapp/${process.cwd()}/node_modules/@agoric/cosmic-swingset/bin/ag-solo`;
+    agSolo = `/usr/src/dapp/node_modules/@agoric/solo/src/entrypoint.cjs`;
```

Comment:
This is at least as correct as it was before, but I didn't realize we were depending upon installation into `/usr/src` in the non-SDK mode.. that's a property of the docker config we use, right?

Eventually I'd like to see these pieces be more standalone. If this is supporting `agoric start` spawning off a solo process, then maybe we can aim towards having it spawn an `ag-solo` process from whereever it might be on $PATH in the future.

Reply:
> I didn't realize we were depending upon installation into `/usr/src` in the non-SDK mode.. that's a property of the docker config we use, right?

Yes that's right, but now that you mention it, `ag-solo` should be in the path as of recent changes to the Dockerfiles.

I'll clean this up.